skyline-vscode/react-ui/src/App.js

```diff
@@ -17,7 +17,11 @@ import MemoryPerfBar from './MemoryPerfBar';
 import ReactTooltip from 'react-tooltip';
 
 
-import { computePercentage, getTraceByLevel } from './utils'; 
+import { computePercentage, getTraceByLevel } from './utils';
+// Changed by John 
```

Comment:
You don't need to wrap this in a comment

Reply:
sure, I will erase the comments